hacks/helm-validate.sh

```diff
@@ -7,4 +7,4 @@ cd "$( dirname "${BASH_SOURCE[0]}" )/.."
 schema_url="https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/"
 k8s_version="1.27.2"
 
-helm kubeconform "./helm/charts/${1}" --strict --schema-location "${schema_url}" -f "hacks/values/${1}.yaml" --kubernetes-version "${k8s_version}" --summary --verbose
+helm kubeconform "./helm/charts/${1}" --strict --schema-location "${schema_url}" --schema-location ./hacks/servicemonitor_v1.json -f "hacks/values/${1}.yaml" --kubernetes-version "${k8s_version}" --summary --verbose
```

Comment:
I think this could be improved, but this is something to be done in a separate issue :)